docs/proposals/20220209-machinepool-machines.md

```diff
@@ -0,0 +1,169 @@
+---
+title: MachinePool Machines
+authors:
+  - "@devigned"
+  - "@jont828"
+  - "@mboersma"
+reviewers:
+  -
+creation-date: 2022-02-09
+last-updated: 2022-02-09
+status: experimental
```

Comment:
shouldn't status be implementable?

Reply:
Ah yes? I guess I thought this "inherited" MachinePool's experimental API status but I will update; it is quite implementable.